main.py

```diff
@@ -48,6 +54,71 @@ def clean(comment: str) -> str:
     return comment.translate(str.maketrans('', '', string.punctuation))
 
 def scrape(link: str) -> ProfessorRating:
+    # Scrape the website, but this time with selenium so we can click the button
+
+    # Initialize headless chromium webdriver
+    options = Options()
+    options.add_argument('-headless')
```

Comment:
You can remove this line if you want to see the webpage actually pop up, but fair warning, it'll take a bit longer because it has to load all the ads. During testing I found out how to add adblock automatically to the webdriver, but funnily enough, adblock and headless are incompatible, so you can only have one or the other. I opted for headless here.